wiki/Mappers_Guild/tr.md

```diff
@@ -0,0 +1,62 @@
+
+Puanlar maplemeyle alakalı ödevleri tamamlayarak kazanılır. Kazanılan miktar ödevler arasında değişiklik gösterir, ancak genellikle harcanması gereken zamana oranlıdırlar. Örneğin, `Hard` bir zorluğu maplemek `Easy` bir zorluğu maplemekten daha uzun zaman alır, bundan ötürü ilki ikinciden daha fazla puan verir.
+
+Eğer beatmap bir göreve bağlıysa zorluk oluşturmak kullanıcıya daha fazla puan kazandırır. Bir görevi bitiş tarihinden önce tamamlamak kullanıcılara maplenen her bir zorluk için bonus puan ve göreve dahil olan tüm kullanıcılara eşit miktarda bonus puan verir. Bu eşit miktardaki bonus bitiş tarihi geçtikten sonra kaybedilir, ancak zorluk bonusları azalan getirileriyle birlikte bitiş tarihinden sonraki 0, 20, ve 40 günlük artışlarıyla korunur.
+
+## Tarihçe
+
+Mappers' Guild 10 mapperdan daha az bir grup ile 2017'nin sonlarına doğru oluşturuldu. O dönemde yeni eklenecek olan bir featured artist, [MOtOLOiD](https://osu.ppy.sh/beatmaps/artists/19), hakkındaki ön bilgilendirme ile sanatçının [haber duyurusuna](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid) eklenmek üzere onlar 5 tane beatmapin oluşturulması için görevlendirildi.
+
+Daha önceki featured artistleri tanıtmak için, grup üye sayısını arttırdı ve farklı sanatçılardan 8 tane beatmap derecelendirdi. Bu beatmapler indirilebilir bir beatmap paketiyle birlikte bir [haber gönderisi](https://osu.ppy.sh/home/news/2018-03-17-new-mappers-guild-pack-release) ile yayımlandı.
+
+Her iki beatmap paketinin ayak izlerini takip ederek, Mappers' Guild, daha eski featured artist şarkılarının paketlerine ek olarak, [High Tea Music](https://osu.ppy.sh/home/news/2018-08-13-new-featured-artist-high-tea-music), [Culprate](https://osu.ppy.sh/home/news/2018-11-21-new-featured-artist-culprate), ve [HyuN](https://osu.ppy.sh/beatmaps/artists/33) gibi, yeni featured artist duyuruları için paketler yayımlarken üye sayısını da sürekli olarak arttırdı.
+
+### Otomasyon
+
+Mappers' Guild için başlangıçtaki plan görevler gibi MMO elementleri bulundurmak olsa da, belirli sayıdaki lonca katılımcıları için bitiş tarihlerini manuel olarak idare etmenin oldukça elverişsiz olduğu belirlendi. 2018'in sonlarına doğru, ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) ve ![][flag_CL] [Milan-](https://osu.ppy.sh/users/1052994) Mappers' Guild sistemini otomatikleştirecek, ödüllerin paylaştırılmasını ve görev/parti sistemlerinin yönetimini daha elverişli hale getirecek bir websitesi geliştirmeye başladı.
+
+Mappers' Guild ile ilgili her şeyin günümüzde kontrol edildiği yer olan [Mappers' Guild websitesi](https://mappersguild.com/), 1 Mart 2019 tarihinde faaliyete geçti. Herkese açık olduğundan dolayı, lonca 500'ün üzerinde bir üye sayısına ulaşmış ve osu! topluluğunun geri kalanını featured artist mapleme içerikleriyle güncel tutmak amacıyla [güncelleme haber gönderilerini](https://osu.ppy.sh/home/news/2019-11-29-featured-artist-beatmap-updates-from-the-mappers-guild) yayımlamaktadır.
```

Comment:
```suggestion
Mappers' Guild ile ilgili her şeyin günümüzde kontrol edildiği yer olan [Mappers' Guild websitesi](https://mappersguild.com/), 1 Mart 2019 tarihinde faaliyete geçti. Herkese açık olduğu için, lonca 500'ün üzerinde bir üye sayısına ulaşmıştır ve osu! topluluğunun geri kalanını featured artist mapleme içerikleriyle güncel tutmak amacıyla [güncelleme haber gönderilerini](https://osu.ppy.sh/home/news/2019-11-29-featured-artist-beatmap-updates-from-the-mappers-guild) yayımlamaktadır.
```

i still don't like the last sentence too much. not sure on a better translation though